acceptance/bundle/run/scripts/cwd/databricks.yml

```diff
@@ -0,0 +1,6 @@
+bundle:
+  name: cheetah
+
+scripts:
```

Comment:
Maybe to make this test even more complete I'd move this scripts section into nested configuration folder so it's not in root to double check that we don't run at where the script is defined